src/schemas/app-manifest/distributor/distributor.json

```diff
@@ -20,10 +20,6 @@
       "$ref": "https://meta.rdkcentral.com/firebolt/app-manifest/distributor/fallback",
       "description": "distributor.fallback: Optional settings to control what should happen if the app can't launch."
     },
-    "deviceSkus": {
-      "$ref": "https://meta.rdkcentral.com/firebolt/app-manifest/distributor/deviceSkus",
-      "description": "distributor.deviceSkus: Optional allowed and/or excluded Device SKUs to control the subset of devices upon which the app may run."
-    },
     "signing": {
```

Comment:
maybe remove signing here too

Reply:
Not sure, tbh. Figured I'd leave it in until we have a conversation about it. If you feel strongly, I can remove and we can always add it back.